### ISSUE TITLE
Unpack: "equilibrium line" and "snow line"

### ISSUE THREAD

**Issue by pbuttigieg** [Vocamp/Virtual-Hackahon-on-Glacier-topic#3]
According to our definitions [here](https://docs.google.com/spreadsheets/d/1_Nm1vj2hM7a3Gewkqkrnkn0zSTGOQK5JH0tGj8PNIU8/edit#gid=32243948&range=13:13), EL is a fiat boundary formed by averaging.

However, as @sjskhalsa [points out](https://docs.google.com/spreadsheets/d/1_Nm1vj2hM7a3Gewkqkrnkn0zSTGOQK5JH0tGj8PNIU8/edit#gid=32243948&range=D13), this is often proxied by the physical snow line. 

It would be interesting to form a design pattern around these sorts of fiat and material entities...

**Comment by Garybc** [comment]
What competency Qs do we want to ask of an EL that the improved model might serve?  The makeup of the SL in an EL "zone"?

**Comment by Garybc** [comment]
If we take the Fiat Object from BFO as a starting point we might be saying that GlacierX is an instance of a material object and has a fiat object part (Equilibrium line) such that for all times t, if GlacierX exists at t then there is is a EL and EL is demarcated from the remainder of GlacierX  by a two-dimensional continuant fiat boundary. 
But it seems to me we want to say more than this.  This  Equilibrium line is a geographic or location concept with a boundary and it is defined in relation to accumulation and ablation zones.  We might say it plays a role of dividing or separating these.  And there is some a physical feature like a snow line associated with it.

**Comment by sjskhalsa** [comment]
yes, EL is a boundary.  SL is used as proxy because it can be detected visually or spectrally. But SL is not the same, and if you get into unpacking it, you have to bring in the concepts like firn line, superimposed ice zone. From Paterson 1981, The Physics of Glaciers:
![fig9 6](https://user-images.githubusercontent.com/10770264/39863063-d868df9c-5445-11e8-9f83-0d2a70a0da82.gif)

**Comment by Garybc** [comment]
SJ,

Interesting "unpacked" details.
I get the impression that these feature on a feature concepts
like EL and SL cannot be adequately dealt with unless we take some time,
like season or "end of the summer" qualifier in the diagram, (and perhaps
event-actions) into account.
So we always have an EL, but where it is changes by seasonal time as does
all the
other zones that help define it.
Do you think we may need to unpack the processes that affect the
zones beyond the accumulation and abrasion labels and the sub-types?

Gary Berg-Cross
